Multisig/UI/UI Library/HideSystemNavigationBarModifier.swift

```diff
@@ -8,6 +8,7 @@
 
 import SwiftUI
 
+#warning("TODO: do we need this modifier?")
 struct HideSystemNavigationBarModifier: ViewModifier {
```

Comment:
this is a convenience to hide and show the navigation bar when going from the main screen with nav bar hidden to child screen with nav bar shown.

Since now we have navbar always shown, this should not be needed anymore